indexing-service/src/test/java/org/apache/druid/indexing/common/task/IndexTaskTest.java

```diff
@@ -254,8 +254,8 @@ public void testCorrectInputSourceTypes() throws IOException
     Assert.assertEquals(
         Collections.singleton(
             new ResourceAction(new Resource(
-                ResourceType.EXTERNAL,
-                LocalInputSource.TYPE_KEY
+                LocalInputSource.TYPE_KEY,
```

Comment:
To avoid any confusion, I think it'd be better to rename all references of the tests that now invoke `getInputSourceResources` (previously `getInputSourceTypes`):
 - `testCorrectInputSourceTypes` -> `testCorrectInputSourceResources`?
- `hasCorrectInputSourceTypes ` -> `hasCorrectInputSourceResources`
...and so on

Reply:
fixed